release/relman.py

```diff
@@ -0,0 +1,498 @@
+#!/usr/bin/env python
```

Comment:
This is an existing tool that was already being used in the `nuodb-control-plane` repo.